crates/pbs/src/routes/register_validator.rs

```diff
@@ -26,6 +32,13 @@ pub async fn handle_register_validator<S: BuilderApiState, A: BuilderApi<S>>(
 
     info!(ua, num_registrations = registrations.len());
 
+    if state.has_monitors() {
+        // send registrations to monitors
+        for relay_monitor in state.pbs_config().relay_monitors.clone() {
+            tokio::spawn(send_relay_monitor_registrations(registrations.clone(), relay_monitor));
```

Comment:
if a task if failing we are not managing the error.
We could do something like
```rust
async fn send_relay_monitor_registrations(
    registrations: Vec<ValidatorRegistration>,
    relay_monitor_url: Url,
) -> eyre::Result<()> {...}
```
And then handling the error like

```rust
let handles: Vec<_> = state.pbs_config().relay_monitors.clone().into_iter().map(|relay_monitor| {
    let registrations_clone = registrations.clone();
    tokio::spawn(async move {
        send_relay_monitor_registrations(registrations_clone, relay_monitor).await
    })
}).collect();

let results: Result<Vec<_>, _> = try_join!(handles.into_iter().map(|h| h));
match results {
    Ok(_) => println!("All tasks completed successfully."),
    Err(e) => eprintln!("A task failed: {:?}", e),
}
```